modules/ROOT/pages/dw-core.adoc

```diff
@@ -26,12 +26,12 @@ attributes in the input object.
 returns only matching objects, filtering out the rest from the output.
 | xref:dw-core-functions-find.adoc[find] | Returns indices of an input that match a specified value.
 | xref:dw-core-functions-flatmap.adoc[flatMap] | Iterates over each item in an array and flattens the results.
-| xref:dw-core-functions-flatten.adoc[flatten] | Turns a set of subarrays (such as [ `[1,2,3], [4,5,[6]], [], [null] ]`) into a single, flattened array (such as `[ 1, 2, 3, 4, 5, [6], null ]`).
+| xref:dw-core-functions-flatten.adoc[flatten] | Helper function that enables `flatten` to work with a `null` value.
```

Comment:
TODO: FIX. Ohhh. We don't want this description used for flatten. Might require attention from MDA.